xml/obs_staging_workflow.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.1" xml:id="cha.obs.stagingworkflow"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" >
+ <title>Staging Workflow</title>
+ <para/>
+ <remark>TBD</remark>
+<!--
+ <sect1>
+  <title>Handling Dependencies</title>
+  <para/>
+  <remark>TBD</remark>
+ </sect1>
+-->
+ <sect1 xml:id="sec.obs.staging.workflow.usage">
+  <title>Working with requests</title>
+  <para>One of the main features of the staging workflow is assigning incoming requests to different staging projects which is described in this section.</para>
+
+  <sect2 xml:id="sec.obs.staging.workflow.assign.request">
+    <title>Assign requests into a staging project</title>
+    <para>
+      Our main project openSUSE:Factory received requests with number 1 and 2. 
+      We would like to group these two requests together and move them into the staging project openSUSE:Factory:Staging:A.
+      This can be done with the following command which will create a link of the source package in openSUSE:Factory:Staging:A.
+    </para>
+    <screen>
+      geeko > <command>osc</command> api -X POST '/staging_project/openSUSE:Factory:Staging:A/staged_requests' -d '&lt;requests&gt;&lt;number&gt;1&lt;/number&gt;&lt;number&gt;2&lt;/number>&lt;/requests&gt;'
```

Comment:
Hm, `osc api` is not really user documentation right? It's apidocs...

Reply:
Well, it's currently the only way we have to assign requests to a staging project:smile:

I think it's valid to mention it in our documentation. Whether it's here or some other guide I don't mind. But let's reference between the different staging related documentation snippets.

Reply:
Well, we have apidocs for this right?

Reply:
I agree that we should document the API routes in our [apidocs](https://build.opensuse.org/apidocs/index). But we also should to refer to it from here, as long as we don't have a UI for this part.
Also the apidocs is somewhat of a low-level documentation. I think it's a good thing to have some high-level / user friendly documentation as well. As long as we don't just duplicate information.

Reply:
But this brings up the point that we didn't document the API routes yet.

I'll add a card to our backlog @openSUSE/obs-frontend